### ISSUE TITLE
Merge the OpenAPI files into a single file

### ISSUE THREAD

**Issue by jyasskin** [GoogleChrome/chromium-dashboard#3613]
Right now, the OpenAPI specification is split into [api.yaml](https://github.com/GoogleChrome/chromium-dashboard/blob/main/openapi/api.yaml) and several `.paths.yaml` and `.schema.yaml` files interlinked by `$ref` references. That's good source hygiene, but unfortunately, it confuses the [VSCode OpenAPI extension](https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi) as described in https://github.com/42Crunch/vscode-openapi/issues/82. We're losing API browsing, as shown in this screenshot (note the lack of detail under "Paths":

![Screenshot of the OpenAPI extension, with empty Paths items](https://github.com/GoogleChrome/chromium-dashboard/assets/83420/fd29e6ac-d171-4df3-9f8a-a279405577a6)

We're also missing code completion and in-editor validation, although I'm not certain if that's the extension breaking or some other part of VSCode being unable to match the `.yaml` file with the right part of the right schema.

I suggest we merge the whole API definition into a single file, and use the extension and other [tools](https://openapi.tools/) to browse it.

**Comment by jcscottiii** [comment]
I verified that the extension does not work in the broken up state. Previously, after I split the files up, I relied on the openapi generator cli to verify if the current schema is valid (by seeing if I could successfully generate the files). But having the IDE tool validate while editing would reduce the guess-and-run cycles.

> I suggest we merge the whole API definition into a single file, and use the extension and other [tools](https://openapi.tools/) to browse it

@jrobbins Any thoughts on consolidating back to one file?
